src/aind_data_schema_models/registries.py

```diff
@@ -0,0 +1,44 @@
+"""Common registries"""
+
+from typing import Annotated, Union
```

Comment:
To make it compatible with python 3.8, should be
`from typing_extensions import Annotated`